submissions/ayodeleayobami/javascript/product.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title> Product Page </title>
+	<link rel="stylesheet" type="text/css" href="product.css">
+</head>
+<body>
+	
+
+<!---------product-sale------------->
+	<section class="product-sale">
+		<div class="container">
+		<div class="title-box">
+			<h1>product list</h1>
+		</div>
+		<div class="search">
+		<form action=""> 
+			<input class="search"  type="text" name="text"
+			   placeholder="search for product" id="search">
+		    <input class="search-button" type="submit" name="submit">
+		</form>
+		</div>
+			<div class="product-box">
+			<div class="product-content">
```

Comment:
according to the task, these items should not be there until the button is clicked please check the task again and confirm. i am assuming that when you load the page, you don't see any products until the button is clicked.